src/sys/windows/uds/listener.rs

```diff
@@ -0,0 +1,23 @@
+use std::io;
+use std::os::windows::io::AsRawSocket;
+use std::path::Path;
+
+use crate::net::{SocketAddr, UnixStream};
+use crate::sys::windows::stdnet as net;
+
+pub(crate) fn bind(path: &Path) -> io::Result<net::UnixListener> {
+    let listener = net::UnixListener::bind(path)?;
+    listener.set_nonblocking(true)?;
+    Ok(listener)
+}
+
+pub(crate) fn accept(listener: &net::UnixListener) -> io::Result<(UnixStream, SocketAddr)> {
+    listener.set_nonblocking(true)?;
+    listener
+        .accept()
+        .map(|(stream, addr)| (UnixStream::from_std(stream), addr))
```

Comment:
I think the accepted stream needs to be set to non_blocking, `from_std` doesn't do that.

Reply:
The stream returned by `accept` inherits properties from the listener socket, including non-blocking. Since this function is crate internal and only called on non-blocking listeners, it should always return non-blocking streams